src/vmm/src/device_manager/persist.rs

```diff
@@ -530,14 +535,20 @@ impl<'a> Persist<'a> for MMIODeviceManager {
                 device,
                 &balloon_state.device_id,
                 &balloon_state.transport_state,
+                interrupt,
                 &balloon_state.device_info,
                 constructor_args.event_manager,
             )?;
         }
 
         for block_state in &state.block_devices {
+            let interrupt =
+                Arc::new(IrqTrigger::new().expect("Could not create interrupt for MMIO device"));
```

Comment:
We unwrap on `IrqTrigger::new()` literally everywhere. Can we instead just unwrap() inside `new()` and have it not return `Result`?

Reply:
done